### DISCUSSION TITLE
Please help us Star Astroid on Github

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#568] [Announcements]
Starring makes it easy to find Astroid. You can see all the repositories and topics you have starred by going to your [stars page](https://github.com/stars).

Starring a repository also shows appreciation to the repository maintainer for their work. Many of GitHub's repository rankings depend on the number of stars a repository has. In addition, [Explore GitHub](https://github.com/explore) shows popular repositories based on the number of stars they have.

### Starring the Astroid is a simple two-step process.

1. On GitHub.com, navigate to the main page of the Astroid. https://github.com/templaza/astroid-framework
2. In the top-right corner of the page, click Star. 
![Screen-Shot-2024-02-16-at-15 04 52](https://github.com/templaza/astroid-framework/assets/3204103/89951776-35d4-4e4b-9cec-2d4ead0d4f86)

And you can see the Star turn Yellow
![Screen Shot 2024-02-16 at 15 05 08](https://github.com/templaza/astroid-framework/assets/3204103/564b32b1-7687-44f7-ba7a-59c877c53532)

Thank you because your contributions will help us improve the quality of Astroid.

Best Regards,
Sonny

**Comment by paurray** [comment]
Done 👍

**Reply by sonvnn** [reply]
Thank you :)

**Comment by Chacapamac** [comment]
I was a good boy — I already Starred Astroid, the best Joomla Framework 🚀🚀🚀🚀👍

**Comment by Giorgi625** [comment]
Also I would recommend you to contact them https://templatejoomla.com there is Astroid framework on this website with template One and Zero but its links point to Joomdev repositories, would be good if you ask them to fix links and also add template Two.
recently one person posted in Joomla group on facebook that he used Astroid on J3 and now because of moving to J5 he was searching for new template, he even did not knew you have took over Astroid. So some more promotion could be useful to get more users.

**Reply by web-eau-net** [reply]
Hi guys,
Thanks for the mention @Giorgi625  
I'm the father of https://templatejoomla.com and I'll be glad to edit / update our informations with the appropriate ones and to add the new Astroid template in the Joomla Templates Directory. Don't hesitate to contact me @sonvnn  :)

**Reply by Chacapamac** [reply]
Good News!

**Reply by sonvnn** [reply]
@web-eau-net Thank you! I am glad to see you here :)

**Reply by web-eau-net** [reply]
You're most welcome @sonvnn !
Feel free to send me at contact-the-team@templatejoomla.com the appropriate project & demo links for the templates Zero, One, and Two and we'll manage the next steps :)

**Comment by sonvnn** [comment]
The Astroid repository has reached 89 stars on Github. Can we reach 100 stars together? Thank you very much. 
Best regards with Love,
Sonny
